marconi-sidechain/README.md

```diff
@@ -1,65 +1,28 @@
 # marconi-sidechain
 
 `marconi-sidechain` is a lightweight chain follower application for the Sidechain project to index and query specific information from the Cardano blockchain.
+The interface for querying the indexed information uses [JSON-RPC](https://www.jsonrpc.org/specification) over HTTP.
 
-## Purpose
+See the [architecture documentation](./doc/ARCHITECTURE.adoc) for more information on how this application was build.
 
-The purpose of ``marconi-sidechain`` is to encapsulate a subset of the indexers provided in [Marconi Chain Index](../marconi-chain-index) into a cohesive set which are required by the Sidechain team and which provides an interface to allow non-Haskell applications to query the indexed information.
-
-## Interface
+## Prerequisites
 
-The interface for marconi-sidechain uses [JSON-RPC](http://www.simple-is-better.org/rpc/#differences-between-1-0-and-2-0) over HTTP built on top of [Marconi Chain Index](../marconi-chain-index/README.md).
+If using `Nix`:
 
-```
-             Running on a single machine                    Internet
-+----------------------------------------------------+
-|                                                    |                  +-------------+
-|   +----------------+                +-----------+  |                  |             |
-|   |                | node-to-client |           |  |     JSON-RPC     |  sidechain  |
-|   |  cardano-node  +----------------+  marconi  +--+------------------+ application |
-|   |                |      IPC       | sidechain |  |       HTTP       |             |
-|   +----------------+                +----+------+  |                  +------------ +
-|                                          |         |
-|                                          |         |
-|                                      +---+----+    |
-|                                      | SQLite |    |
-|                                      +--------+    |
-|                                                    |
-+----------------------------------------------------+
-```
+* [Nix](https://nixos.org/download.html) (`>=2.5.1`)
+  * Enable [IOHK's binary cache](https://iohk.zendesk.com/hc/en-us/articles/900000673963-Installing-Nix-on-Linux-distribution-and-setting-up-IOHK-binaries) or else you will build the world!
```

Comment:
What's the current status of the cache are we back to IOG's cache or should we still use Moritz' one?

Reply:
For the time being, the most up-to-date one seems to be zw3rk 🤷

Reply:
Updated to point to our CONTRIBUTION.adoc file which specifies the need to use `zw3rk`.